pkgs/development/compilers/elm/default.nix

```diff
@@ -7,8 +7,8 @@
 let
   fetchElmDeps = pkgs.callPackage ./fetchElmDeps.nix { };
 
-  # Haskell packages that require ghc 8.10
-  hs810Pkgs = self: pkgs.haskell.packages.ghc810.override {
+  # Haskell packages that require ghc 9.2
+  hs96Pkgs = self: pkgs.haskell.packages.ghc92.override {
```

Comment:
This is named `hs96Pkgs` (implying ghc 9.6), but it uses ghc 9.2.  Is that a typo?  I see there's also already an `hs92Pkgs` defined below which has been used for elm-format.  I guess we probably will have to keep these two ghc package sets separate to avoid other dependency conflicts, even if they are actually the same ghc version?

Reply:
This is likely bug caused by some refactoring I was attempting to do before squashing. It's likely also the caused for CI failure.